### DISCUSSION TITLE
Проблема с версткой?

### DISCUSSION THREAD

**Discussion by M3teora** [totumonline/totum-ru-issues-and-discussions#543] [ОШБК!] [answered]
Добрый день! Столкнулся с таким моментом. 
Когда поле типа **select** в хэдере, кнопка добавления значения в таблицу из которой берётся этот **select** отрисовывается хорошо (скриншот 1). 
Но когда такое же поле в табличной части, то при добавлении строки, синего плюса я не наблюдаю (скриншот 2). Но можно нажать на край справа (скриншот 3) и тогда функционал сработает (скриншот 4) 

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/7896517/9e3be919-daff-471e-bc31-dd1990f549a4)

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/7896517/c73ece67-4816-4b25-8555-036df80a5112)

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/7896517/cd8f3cfb-ce4c-4efb-bc11-040bcfd4a795)

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/7896517/8d90b1a1-7c56-4270-9d71-dee777256b50)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Мне кажется это уже исправляли. У вас точно обновлено до последней версии?

**Reply by M3teora** [reply]
Верно, обновил и проблема ушла.
